source/03-components/icon/icon.stories.jsx

```diff
@@ -7,7 +7,18 @@ import data from './icon.yml';
 
 const settings = {
   title: 'Components/Icon',
-  decorators: [withGlobalWrapper],
+  decorators: [
+    (Story, { args }) => {
+      const icons = globalData.icons.map(icon => (
+        <p key={icon}>
+          <Story args={{ ...args, icon_name: icon }} />
+          &nbsp;&nbsp;{ icon }
+        </p>
+      ));
+      return <>{icons}</>;
+    },
+    withGlobalWrapper,
+  ],
```

Comment:
This can be shortened a bit to
```
    (Story, { args }) => globalData.icons.map(icon => (
        <p key={icon}>
          <Story args={{ ...args, icon_name: icon }} />
          &nbsp;&nbsp;{ icon }
        </p>
      ));
    },
```

I do think this one makes sense with a decorator.

Reply:
Updated.